packages/p2p/src/pages/my-ads/edit-ad-summary.jsx

```diff
@@ -34,8 +35,8 @@ const EditAdSummary = ({ offer_amount, price_rate, type }) => {
 
     if (offer_amount) {
         const components = [
-            <Text key={0} weight='bold' size='xs' color='status-info-blue' />,
-            <Text key={1} weight='normal' size='xs' color='status-info-blue' />,
+            <Text key={0} className='edit-ad-summary' weight='bold' size='xs' color='status-info-blue' />,
+            <Text key={1} className='edit-ad-summary' weight='normal' size='xs' color='status-info-blue' />,
```

Comment:
```suggestion
            <Text key={1} className='edit-ad-summary' size='xs' color='status-info-blue' />,
```

here as well can remove the weight='normal'